NOTES.VALGRIND

```diff
@@ -51,20 +51,20 @@ used to control what capabilities OpenSSL uses.
 As of valgrind-3.15.0 on Linux/x86_64, instructions up to AVX2 are
 supported. Setting the following disables instructions beyond AVX2:
 
-   OPENSSL_ia32cap=":0"
+    OPENSSL_ia32cap=":0"
 
 This variable may need to be set to something different based on the
 processor and Valgrind version you are running tests on. More information
-may be found in docs/man3/OPENSSL_ia32cap.pod.
+may be found in [docs/man3/OPENSSL_ia32cap.pod](docs/man3/OPENSSL_ia32cap.pod).
 
-Additional variables (i.e. VERBOSE and TESTS) are described in the
-INSTALL file in the root of the OpenSSL source tree.
+Additional variables (such as `VERBOSE` and `TESTS`) are described in the
+file [test/README.md]/(test/README.md).
```

Comment:
Spurious /

Also, there shouldn't be a need to specify the URI in round brackets.

Reply:
Yes :)